src/modules/apps/utils.ts

```diff
@@ -125,6 +125,13 @@ export const appLatestVersion = (app: string, version = 'x'): Promise<string | n
     .catch(handleError(app))
 }
 
+export const appIdFromRegistry = (app: string, majorLocator: string) => {
```

Comment:
*nit*: i think is't a little weird an util method which create a client and perform a http request. I suggest keeping it only where it's used and create a util method only to prop the id if you think it's necessary.